src/commands/mock.ts

```diff
@@ -0,0 +1,51 @@
+import { CommandInteraction, SlashCommandBuilder } from 'discord.js';
+import { mockTargets } from '../index';
+
+export const Mock = {
+  info: new SlashCommandBuilder()
+    .setName('mock')
+    .setDescription("Mocks designated member")
+    .addUserOption(option =>
+        option.setName('member')
+            .setDescription('Choose which member you want to be mocked')
+            .setRequired(true)),
+    
+    run: async (interaction: CommandInteraction): Promise<void> => {
+        if (!interaction.guild){ //checks if the command is NOT done in a server
+            await interaction.reply(`This command can only be done in a server.`);
+            return;
+        }
+            //Looks for "staff" role in server      
+          const role = interaction.guild.roles.cache.find(role => role.name === "staff");
+
+          if (!role) { 
+            await interaction.reply(`There is no "staff" role on this server. + 
+                                     Contact a developer for more information.`);
```

Comment:
I would do something like
```ts
await interaction.reply("There is no \"staff\" role on this server."
    + " Contact a developer for more information."); 
```

Here, we're using the backslash to ensure that the quotation mark used inside the string should be treated as a character in the string, not the end of the string. This is known as an _escape_ sequence. Wikipedia has a good table here: https://en.wikipedia.org/wiki/Escape_sequences_in_C. 

Feel free to ask me any questions about this.